src/index.ts

```diff
@@ -70,6 +70,7 @@ export async function analyze(): Promise<void> {
 
   // Collect default options: Env and listeners
   const PATH = process.env.PATH || '';
+  // eslint-disable-next-line @typescript-eslint/naming-convention
   const defaultOptions = { env: { ...process.env, PATH, FOSSA_API_KEY }, listeners };
```

Comment:
I think camelCase object literal properties is a good rule in general except obviously in this case where we're specifying env vars, so I'm disabling the lint for this line (as well as the other instance later in this file).